pkgs/applications/window-managers/cagebreak/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchFromGitHub
+, meson, ninja, pkg-config, wayland, scdoc, makeWrapper
+, wlroots, wayland-protocols, pixman, libxkbcommon
+, cairo , pango, fontconfig, pandoc, udev
+, xwayland ? null
+, nixosTests
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cagebreak";
+  version = "1.4.2";
+
+  src = fetchFromGitHub {
+    owner = "project-repo";
+    repo = "cagebreak";
+    rev = version;
+    sha256 = "+Ww1rsUR7qe/BixLPR8GiRc3C6QmpLzWpT2wym8b4/M=";
```

Comment:
you can actually use `hash` in `fetchFromGitHub`

Reply:
@worldofpeace is this some new policy?

Reply:
The hash here is encoded in base64, which is the default format in the next nix version. I'm assuming the author is actually using this. In nix we've added SRI hashes, which looks like
```
 hash = "sha256-NrhcUFQM+L66KaDRRpAoC+z5s54a+1fqEepTRXVZ5Qs=";
```

Currently nixpkgs is a mix of both of these, as there's no official precedent to move all of the hashes to the new format. Likely when Nix 3.0 is released nixpkgs will change over incrementally.